src/visitPage.js

```diff
@@ -95,7 +95,7 @@ class VisitPage extends Component {
 
   setClientProperty = (e, data) => {
     let propertyName = data.name
-    let value = (data.checked != null && data.checked != undefined) ? data.checked : data.value  //This handles both complex values and checked values
+    let value = (data.checked !== null && data.checked !== undefined) ? data.checked : data.value  //This handles both complex values and checked values
```

Comment:
You made an excellent point and I have no idea what I was thinking. With that in mind, though, maybe it'd be best to just check for a boolean.